request-builder.js

```diff
@@ -9,6 +9,9 @@ function buildSlots(slots) {
       name: key,
       value: value
     };
+    if(! _.isString(value)){
+      res[key] = {...res[key],...value}
```

Comment:
What is the end value you'd like `res[key]` to hold? Assuming `value` is an object, wouldn't this code produce an object where its properties would be duplicated, once inside the `value` property of the resulting object and again the same properties spread one level higher? Is that what you want? If so, why?

```js
{
  name: 'key',
  value: { /* value props */ 
     value1: ...,
    value2: ...,
  }
  value1: ..., // Same value props already in `value`, but spread out one level higher
  value2: ...
}
```

Reply:
Thank you for your quick reply. And sorry for my poor PR.
What I want to do is not to set an object to value props, but to set spread value.

So expected result is bellow.
```
//userSays('Foo',{key:{value:'value',value1:'value1',value2:'value2'}})
{
  name: 'key',
  value:'value',
  value1:'value1',
  value2: 'value2',
}
```
But my PR is hard to understand, so I rewrote it to show intension clearly.